docs/migration-guides/prometheus-migration.md

```diff
@@ -144,7 +143,16 @@ After deploying the generated manifest you will have a *rsync* container that yo
 
 > Note: Make sure you change the placeholders NAMESPACE and PROMETHEUS_POD_NAME for the Kubernetes namespace and Prometheus pod name you have. For example: `kubectl exec -it -n monitoring prometheus-d776d68fb-vc7gt -c sidecar-rsync sh`
 
-Check also that a new service has been created for Prometheus, as shown in the example command and output below:
+You should now expose the rsync service running on the recently added container.
+
+Obtain the Prometheus pod name:
+
+```bash
+~ $ POD=$(kubectl -n NAMESPACE get pods -l name=prometheus -o NAME)
+~ $ kubectl expose -n NAMESPACE $POD --name=prometheus-rsync --port=873
```

Comment:
Thanks for adding this.
It seems I forgot to add the service block. Could you update the jsonnet with this?

```
// target-add-rsync.jsonnet: add rsync sidecar container, exposing
// <volume_name> below via rsync, by overloading saved "target.json"

// NOTE: We're not assuming a particular instrumentation tool (e.g ksonnet, kubecfg via jsonnet)
// but rather doing an override from original sts / deploy output json

// 1) Save existing deployment/sts to target.json:
//   kubectl get deployments -n NAMESPACE NAME -o target.json
local target = import "target.json";

// volume_name must be the name of existing volume name inside target's podSpec
local volume_name = "data";

// 2) show/diff/update resulting composition ('-J lib' where lib is the path to kube.libsonnet)
//   kubecfg -J lib show target-add-rsync.jsonnet

// For testing purposes, You can fake above by creating a deploy with an emptyDir volume:
//    kubectl run --image=nginx:latest --overrides '{"spec":{"template":{"spec":{ "volumes": [{"name": "data", "emptyDir": {}}]}}}}' nginx-tgt
//    kubectl get deploy nginx-tgt -ojson > target.json

local kube = import "kube.libsonnet";
local prometheus_container_name = "prometheus";
local rsync_conf_b64 = std.base64(
  |||
    [data]
    path = /data
    list = false
    use chroot = false
    read only = true
  |||
);

local rsync_container_name = "sidecar-rsync";

// sidecar-rsync container (mounting <volume_name>) to be exposed via rsync
local rsync_container = kube.Container(rsync_container_name) {
  image: "alpine:3.8",
  args: [
    "sh",
    "-c",
    |||
      apk update && apk add rsync
      echo '%s' | base64 -d > /rsync.conf
      rsync -vvv --daemon --no-detach --config=/rsync.conf
    ||| % [rsync_conf_b64],
  ],
  volumeMounts_+: {
    [volume_name]: { mountPath: "/data" },
  },
};

local containers_obj(containers) = {
  [container.name]: container for container in containers
};

local stack = {
  // Create a <NAME>-rsync service to expose rsync
  rsync_svc: kube.Service($.target_with_sidecar.metadata.name + "-rsync") {
    metadata+: { namespace: $.target_with_sidecar.metadata.namespace },
    spec: {
      selector: $.target_with_sidecar.spec.template.metadata.labels,
      ports: [{ name: "rsync", port: 873, targetPort: self.port }],
      type: "ClusterIP",
    },
  },
  target_with_sidecar: target {
    spec+: {
      template+: {
        spec+: {
          // convert containers array to obj for easier jsonnet
          // handling: in particular, if we just added the new container
          // to existing containers[] array, we wouldn't be able to re-run
          // this, as we'd get duplicated containers.
          // By referring the added container via rsync_container_name, we
          // can easily override it.

          local containers = super.containers,
          containers: kube.PodSpec {
            default_container:: containers[0].name,
            // create or override(existing) rsync_container
            containers_+: containers_obj(containers) {
              [rsync_container_name]: rsync_container,
              [prometheus_container_name]+:{
                args+: [
                  "--web.enable-admin-api",
                ],
              },
            },
          }.containers,
        },
      },
    },
  },
};

kube.List() { items_+: stack }
```

With that it shouldn't be necessary to add the `kubectl expose ... `

Note: This comment doesn't have included Gus feedback